tic_tac_toe/game.py

```diff
@@ -63,22 +79,68 @@ def _check_winning_combinations(board, player):
     Returns the player (winner) of any of the winning combinations is completed
     by given player, or None otherwise.
     """
-    pass
-
-
+    
+    def check_diags(board, player):
+        diag1=[board[i][i] for i in range(3)]
+        diag2 = []
+        diag2.append(board[0][2])
+        diag2.append(board[1][1])
+        diag2.append(board[2][0])
+        if diag1.count(player) == 3 or diag2.count(player) == 3:
+            return player
+        return None
+    
+    
+    def check_rows(board, player):
+        rows = [board[i][j] for i in range(3) for j in range(3)]
+        if rows[:3].count(player) == 3 or rows[3:6].count(player) == 3 or rows[6:9].count(player) == 3:
+            return player
+        return None
+        
+    def check_columns(board, player):
+        cols = [board[j][i] for i in range(3) for j in range(3)]
+        if cols[:3].count(player) == 3 or cols[3:6].count(player) == 3 or cols[6:9].count(player) == 3:
+            return player
+        return None
+    
+    if check_diags(board, player):
+        return player
+    elif check_rows(board, player):
+        return player
+    elif check_columns(board, player):
+        return player
+    else:
+        return None
+    
 # public interface
 def start_new_game(player1, player2):
     """
     Creates and returns a new game configuration.
     """
-    pass
+    
+    return {
+        'player1':player1,
+        'player2':player2,
+        'board': [
+            ["-", "-", "-"],
+            ["-", "-", "-"],
+            ["-", "-", "-"]
+        ],
+        'next_turn': "X",
+        'winner': None
+        }
 
 
 def get_winner(game):
     """
     Returns the winner player if any, or None otherwise.
     """
-    pass
+    if _check_winning_combinations(game['board'], game['player1']):
+        return game['player1']
+    elif _check_winning_combinations(game['board'], game['player2']):
+        return game['player2']
+    else:
+        return None
```

Comment:
I'm not sure because I don't thing there is game['player'], only game['player1'] and game['player2'].

I'm not sure I understand exactly what you mean, could you explain it a little more?

BTW: Thanks for checking my code out!